src/scripting/game_lua_kernel.cpp

```diff
@@ -2255,7 +2255,13 @@ int game_lua_kernel::intf_print(lua_State *L) {
 
 	int size = cfg["size"].to_int(font::SIZE_SMALL);
 	int lifetime = cfg["duration"].to_int(50);
-	SDL_Color color = create_color(cfg["red"], cfg["green"], cfg["blue"]);
+
+	SDL_Color color = font::LABEL_COLOR;
```

Comment:
Is this a one of the pango colours? Because pango uses a different colour encoding than the rest of our code.

Wesnoth colours are packed into a 32-bit integer as ARGB, pango as RGBA.